### PR TITLE
GSLUX-620: Traductions manquantes pour les layers

### PR DESCRIPTION
**Pull request by AlitaBernachot**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-620

### Description

Translation files (for all lang) need an update.
Some translations are missing, especially for new layers, see image below:

image (2) (1)

### Done:

- [x] Updated all client.json (x4)
- [x] Fix i18n key for layer panel title (`"layers"` instead of `"Layers"`)
- [x] Added `<h1>` for accessibility
- [x] Fix missing translation when bg layer is blank in prod mode build for lib (in dev, empty translation - eg. `"My translation": ""` - is replaced by the key, in prod, empty translation is kept)

**Edit**
- [x] Updated themes fixtures, now catalog is up to date

### CODE REVIEW

File: bundle/assets/locales/client.de.json [hunk 1]
```diff
@@ -1721,5 +1721,6 @@
   "Buildings": "Gebäude",
   "Water": "Hydrologie",
   "Arrière-fond": "Hintergrund",
-  "Hillshade": "Relief"
+  "Hillshade": "Relief",
+  "main_culture": "Culture"
```
Comment (mki-c2c) [inline bundle/assets/locales/client.de.json:1725]: does not seem like a correct German translation, where is the source for it ?

File: public/assets/locales/client.de.json [hunk 2]
```diff
@@ -1721,5 +1721,6 @@
   "Buildings": "Gebäude",
   "Water": "Hydrologie",
   "Arrière-fond": "Hintergrund",
-  "Hillshade": "Relief"
+  "Hillshade": "Relief",
+  "main_culture": "Culture"
```
Comment (mki-c2c) [inline public/assets/locales/client.de.json:1725]: does not seem like a correct German translation, where is the source for it ?

**Review by mki-c2c — APPROVED** [review]
for my information, how do you extract the translation files from the v3 deployment ?
Is it manual or automatic extraction ?

**Comment by mki-c2c** [comment]
the PR breaks the V4 demo app, the themes fixture does not seem to be in line with the translations.

**Review by mki-c2c — CHANGES_REQUESTED** [review]
Not sure if the catalog/themes fixture and the translations are in line... To be discussed

File: public/assets/locales/client.de.json (hunk 2 above)
Comment (AlitaBernachot) [reply public/assets/locales/client.de.json]: Thanks for reporting this, I have made a wrong copy/past. Now the client.de.json is up to date.
All translations files have been updated with a manual copy of static files from prod eg. https://map.geoportail.lu/static/7ab2ad3b3c504ca7ba46f9cdec201f72/de.json?cache=7ab2ad3b3c504ca7ba46f9cdec201f72

**Review by mki-c2c — APPROVED** [review]
Thanks for the fix, I can also see from your comment how to recover the translation data from prod server.

**Pull request merged**